cloud/logger/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@raincatcher/logger",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Comment:
I just do not want to waste time for bumps. If we release this sprint we can decide what version we need. For the moment best to keep versions the same